src/components/BookingForm.vue

```diff
@@ -11,19 +30,22 @@
 </template>
 
 <script>
-import BadContainedButton from "../components/BadContainedButton";
+import BadContainedButton from "@/components/BadContainedButton";
 export default {
   name: "BookingForm",
   data() {
-    return {};
+    return {
+      officeId: "",
+      bookingdate: "",
```

Comment:
I would suggest to use camelCase for all variables

Reply:
These values are gonna be sent to back end, I assumed that what they would expect since it was coded like when we branched out